data/json/mapgen/musicstore.json

```diff
@@ -0,0 +1,144 @@
+                "zs______,______,______sz",
+                "ssssssssssssssssssssssss",
+                "_--5555--5555-++--55555_",
+                "_|^.CC....CC.....#.EcC8_",
+                "_8...............#....|_",
+                "_8E..###rr...rr..#....8_",
+                "_8c..###rr...rr..####.8_",
+                "_|...........rr.......|_",
+                "_|...######..........C|_",
+                "_|...######...........|_",
+                "_8...........|+-|+-|+-|_",
+                "_8E..rrrrrr..|..|..|..8_",
+                "_8c..rrrrrr..|..|..|..8_",
+                "_|...........|Ec|Ec|Ec|_",
+                "_|D---D|D-55----------|_",
+                "_|.S|S.|ssssssszz_z_____",
+                "_|T#|#T|________________",
+                "_-H---H-________________",
+                "________________________"
+            ],
+            "terrain": {
+                "_": "t_pavement",
+                ",": "t_pavement_y",
+                "s": "t_sidewalk",
+                "-": "t_wall_h",
+                "|": "t_wall_v",
+                "5": "t_wall_glass_h",
+                "8": "t_wall_glass_v",
+                ".": "t_linoleum_gray",
+                "+": "t_door_glass_c",
+                "D": "t_door_c",
+                "L": "t_door_locked",
+                "H": "t_curtains",
+                "w": "t_linoleum_gray",
+                "y": "t_linoleum_gray",
+                "%": "t_shrub",
+                "^": "t_linoleum_gray",
+                "r": "t_linoleum_gray",
+                "l": "t_linoleum_gray",
+                "#": "t_linoleum_gray",
+                "E": "t_linoleum_gray",
+                "t": "t_linoleum_gray",
+                "C": "t_linoleum_gray",
+                "c": "t_linoleum_gray",
+                "F": "t_linoleum_gray",
+                "S": "t_linoleum_gray",
+                "T": "t_linoleum_gray",
+                "d": "t_pavement",
+                "z": "t_shrub"
+            },
+            "furniture": {
+                "_": "f_null",
+                ",": "f_null",
+                "s": "f_null",
+                "-": "f_null",
+                "|": "f_null",
+                "5": "f_null",
+                "8": "f_null",
+                ".": "f_null",
+                "+": "f_null",
+                "D": "f_null",
+                "L": "f_null",
+                "H": "f_null",
+                "W": "f_null",
+                "%": "f_null",
+                "x": "f_null",
+                "^": "f_indoor_plant",
+                "r": "f_rack",
+                "#": "f_counter",
+                "E": "f_desk",
+                "C": "f_displaycase",
+                "c": "f_chair",
+                "S": "f_sink",
+                "d": "f_dumpster",
+                "z": "f_null"
+            },
+            "toilets": {
+                "T": { }
+            },
+            "add": [
+                { "item": "violin_golden", "x": 21, "y": 8, "chance": 95}
```

Comment:
From my playtesting, it seemed like the higher the number, the less likely a spawn. I believe that means that there's only a 5% chance to spawn.